.github/workflows/release-kvx-llvm.yml

```diff
@@ -0,0 +1,68 @@
+name: Release LLVM
```

Comment:
For the release script I simply copy/pasted the one for gcc. I removed build for ubuntu16 since it is deprecated and removed soon. 

We could probably merge the two scripts (one for gcc and llvm) but it seems a bit complicated since they don't have the same tags name rule for CI trigger.